src/gateway/client/dispatch.rs

```diff
@@ -206,7 +195,7 @@ fn update_cache_with_event(
             }
         },
         Event::GuildDelete(mut event) => {
-            let full = if_cache!(event.update(cache));
+            let full = if_cache!(update_cache!(cache, event));
```

Comment:
Why change this? I'm fine with merging as-is, just curious.

Reply:
Simply because `update_cache!` is used elsewhere in the file to do the same thing. Figured I'd make it consistent.